includes/class-wc-payments-platform-checkout-button-handler.php

```diff
@@ -48,6 +48,39 @@ public function __construct( WC_Payments_Account $account, WC_Payment_Gateway_WC
 		add_action( 'init', [ $this, 'init' ] );
 	}
 
+	/**
+	 * Indicates eligibility for WooPay via feature flag.
+	 *
+	 * @var bool
+	 */
+	private $is_platform_checkout_eligible;
+
+	/**
+	 * Indicates whether WooPay is enabled.
+	 *
+	 * @var bool
+	 */
+	private $is_platform_checkout_enabled;
+
+	/**
+	 * Indicates whether WooPay express checkout is enabled.
+	 *
+	 * @var bool
+	 */
+	private $is_platform_checkout_express_button_enabled;
+
+	/**
+	 * Indicates whether WooPay and WooPay express checkout are enabled.
+	 *
+	 * @return bool
+	 */
+	public function is_woopay_enabled() {
+		if ( ! ( $this->is_platform_checkout_eligible && $this->is_platform_checkout_enabled && $this->is_platform_checkout_express_button_enabled ) ) {
+			return false;
+		}
+		return true;
```

Comment:
It looks like this could be simplified:
```suggestion
		return $this->is_platform_checkout_eligible && $this->is_platform_checkout_enabled && $this->is_platform_checkout_express_button_enabled;
```

Reply:
Yes, that looks much better! https://github.com/Automattic/woocommerce-payments/pull/5520/commits/ba2e62020f30d1383f86f70a26f5d1a029b77368